### PR TITLE
Missing Indexes Added for IS-5.8

### PR DESCRIPTION
**Pull request by Buddhimah**
### Proposed changes in this pull request

Newly added Indexes were added to the IS DBScrips 
Related Issues:
wso2/product-is#4891
wso2/product-is#5031

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql [hunk 1]
```diff
@@ -1430,3 +1430,11 @@ CREATE INDEX IDX_USER_ID ON IDN_AUTH_USER_SESSION_MAPPING (USER_ID)
   /
 CREATE INDEX IDX_SESSION_ID ON IDN_AUTH_USER_SESSION_MAPPING (SESSION_ID)
   /
+
+  -- IDN_OAUTH_CONSUMER_APPS --
+CREATE INDEX IDX_OCA_UM_TID_UD_APN ON IDN_OAUTH_CONSUMER_APPS(USERNAME,TENANT_ID,USER_DOMAIN, APP_NAME)
```
Comment (mefarazath) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql]: there should be another index targeting the query SELECT PROPERTY_KEY, PROPERTY_VALUE FROM IDN_OIDC_PROPERTY WHERE TENANT_ID = ? AND CONSUMER_KEY = ? right?

Comment (Buddhimah) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql]: added

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle.sql [hunk 2]
```diff
@@ -1437,3 +1437,15 @@ CREATE INDEX IDX_USER_ID ON IDN_AUTH_USER_SESSION_MAPPING (USER_ID)
...
+CREATE INDEX IDX_OCA_UM_TID_UD_APN ON IDN_OAUTH_CONSUMER_APPS(USERNAME,TENANT_ID,USER_DOMAIN, APP_NAME)
+/
+
+-- IDX_SPI_APP --
+CREATE INDEX IDX_SPI_APP ON SP_INBOUND_AUTH(APP_ID)
+/
+
+-- IDN_OIDC_PROPERTY --
+CREATE INDEX IDX_IOP_TID_CK ON IDN_OIDC_PROPERTY(TENANT_ID,CONSUMER_KEY)
+/
```
Comment (mefarazath) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle.sql]: let's add a newline at EOF

**Comment by mefarazath** [comment]
@Buddhimah have we captured these indexes in the migration client?

**Comment by JKAUSHALYA** [comment]
Please check the following 2 issues as well. We can get some from there related to identity tables. 
https://github.com/wso2/product-is/issues/3916
https://github.com/wso2/product-is/issues/4891

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle.sql (hunk 2 above)
Comment (Buddhimah) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle.sql]: done

**Comment by Buddhimah** [comment]
wso2/product-is#3916
wso2/product-is#4891 
These issues were deprioritized

**Pull request merged**